<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsDialog</class>
 <widget class="QDialog" name="settingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="informer">
    <attribute name="title">
     <string>Informer</string>
    </attribute>
    <widget class="QGroupBox" name="soundGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Sound</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="soundPathEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>301</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="soundPathLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Path to wav file:</string>
      </property>
     </widget>
     <widget class="QToolButton" name="soundPathToolButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>26</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="soundErrorLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>151</red>
            <green>151</green>
            <blue>151</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="soundCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Play sound</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="messageGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>481</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Message</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLineEdit" name="patternLineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>301</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="patternLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Message template:</string>
      </property>
     </widget>
     <widget class="QLabel" name="patternInfLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>461</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>%1 - cabinet, %2 - date, %3 - time, %4 - notes</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="messageCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>461</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Show message</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="DB">
    <attribute name="title">
     <string>Datebase</string>
    </attribute>
    <widget class="QLineEdit" name="dbPathLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>331</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="dbTableLineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>331</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="dbToolButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>26</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="dbLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Path to date base:</string>
     </property>
    </widget>
    <widget class="QLabel" name="dbTableLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Table name:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
